frontend/app/common/api.ts

```diff
@@ -150,7 +150,7 @@ export const deleteMe = (): Promise<{
 
 export const approveDeleteMe = (token: string): Promise<void> =>
   fetcher.get({
-    url: `/admin/deleteme?token=${token}`,
+    url: `/admin/deleteme?token=${encodeURIComponent(token)}`,
```

Comment:
is it really needed? jwt supposed to be encoded already and token is jwt

Reply:
yep, seems like overprotection from my side